docs/extend/extend-zowe-overview.md

```diff
@@ -1,56 +1,60 @@
 
-The API Mediation Layer includes an API gateway that acts as a reverse proxy server through which API requests can be routed from clients on its northbound edge to z/OS servers on its southbound edge. The API gateway is extensible so you can add REST APIs for z/OS servers to its list of services. For information about how to onboard REST APIs, see [Onboarding Overview](extend-apiml/onboard-overview.md).  
+## Extend Zowe API Mediation Layer 
 
-To register a z/OS server with the API Mediation layer, there are two techniques:
-- [dynamic](#dynamic-api-registration)
-- [static](#static-api-registration)
+Zowe API Mediation Layer extenders can build and onboard additional API services to the API ML. REST APIs can register with the API Mediation Layer, which makes them available in the API Catalog and for routing through the API Gateway. For information about how to onboard REST APIs, see the [Onboarding Overview](extend-apiml/onboard-overview.md). 
+
+To register a z/OS service with the API Mediation Layer, there are two approaches:
+- [Dynamic API registration](#dynamic-api-registration)
+- [Static API registration](#static-api-registration)
 
 ### Dynamic API registration
 
-The API Gateway can be called by the server that wants to register their REST APIs through a set of API calls to the API Gateway itself.  To do this, the z/OS server needs to know where the API Gateway is and make the API calls to register or unregister itself.  This knowledge can either be within the z/OS server itself, or more typically is done by introducing a micro service whose task is to register to the API Mediation Layer on behalf of an existing z/OS Service and act as a registration broker.  The coding pattern for the micro service is to create a Java Spring Boot server. For more information, see [Onboarding a Spring Boot based REST API Service](extend-apiml/onboard-spring-boot-enabler.md).  This is a bottom up registration, where the z/OS service beneath the API Mediation Layer is calling up into it to say it is ready to receive API requests as well as information for how it should be rendered on the API catalog.  
+Service registration of a REST API to the API ML is performed through a call to the Discovery Service component of API ML which contains registration data and metadata for that service. Registration requires the z/OS service to know the web address of the API Discovery Service. When Dynamic registration is performed, the service that performs the registration must periodically send heartbeat requests for each registered service instance to the Discovery Service. These requests enable the Discovery Service to monitor the availability of registered service instances. Services that are registered dynamically display the status of the service in the API Catalog after initial service registration. Services that are registered statically also appear in the API Catalog, but the status of service instances is not displayed. 
```

Comment:
Registration of a REST API service to the API ML is performed through a call to the Discovery Service by sending registration data and metadata for the service being registered. Registration requires that the z/OS service must know the web address of the API ML Discovery Service. When Dynamic registration is performed, the service that performs the registration must periodically send to the Discovery Service heartbeat requests for each registered service instance. The heartbeat requests serve to renew the corresponding service instance registration with API ML.